witnet.toml

```diff
@@ -14,10 +14,10 @@ enabled = true
 server_address = "127.0.0.1:21338"
 
 [consensus_constants]
-checkpoints_period = 30
-checkpoint_zero_timestamp = 1584111600
-bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e30"
-genesis_hash = "7a7a438b857e8a50fb3281df32b34f8acd930da8f0d51f308903c8a55e6ff7a9"
+checkpoints_period = 45
+checkpoint_zero_timestamp = 1584748800
+bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e31"
+genesis_hash = "53cf1a851e686292438ebec48e8c502f58d4cca9013e4fed3db114be0cb40d23"
 mining_replication_factor = 1
 activity_period = 240
```

Comment:
@devs is this `activity_period` right for T7.1?

Reply:
No, currently is 2000 by default so if you delete that line it will be fine